Fix Windows version number

peterbozso opened the pull request.

Uruk committed:
Fix Windows version number

msftclas commented:
This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. We will now review your pull request.
_Thanks,
Microsoft Pull Request Bot_

Karl-Bridge-Microsoft committed:
Minor revision to customer suggestion

No need for version number as it will just have to be updated again at some point.

Karl-Bridge-Microsoft reviewed (approved):
Thanks for your contribution to the docs! It's greatly appreciated.
